deploy/sample/ml-config.sample.xml

```diff
@@ -198,6 +198,16 @@
       <fields>
         <field>
           <field-name></field-name>
+          <metadata/>
+          <field-value-searches>false</field-value-searches>
+          <field-value-positions>false</field-value-positions>
+          <word-lexicons/>
+          <included-elements/>
+          <excluded-elements/>
+          <tokenizer-overrides/>
+        </field>
```

Comment:
Hmm, interesting. Does a metadata field take tokenizers, lexicons and such? Looks like not. Perhaps remove those elems from the sample?